.dockerignore

```diff
@@ -3,7 +3,7 @@
 !requirements.txt
 !requirements-dev.txt
 !posthog
-!plugins
+!plugin-server
```

Comment:
Not sure of the implications of this - e.g. when developing plugins w/ docker, will this cause issues?

Reply:
This was added to make sure we _can_ develop with Docker actually.